delegation-toolkit/tutorials/create-custom-caveat-enforcer.md

```diff
@@ -8,24 +8,31 @@ import TabItem from "@theme/TabItem";
 
 # Create a custom caveat enforcer
 
-When [restricting a delegation](restrict-delegation.md), the MetaMask Delegation Toolkit provides some [out-of-the-box caveat enforcers](../../reference/caveats.md)
-that cover common use cases.
-For more granular or custom control, you can follow the instructions on this page to create custom caveat enforcers from scratch.
+This tutorial walks you through creating a custom [caveat enforcer](../concepts/caveat-enforcers.md) and applying it to a [delegation](../concepts/delegation.md).
+
+The MetaMask Delegation Toolkit provides some [out-of-the-box caveat enforcers](../reference/caveats.md) that define rules and restrictions for common use cases.
```

Comment:
Suggestion, something like, "The MetaMask Delegation Toolkit includes [built-in caveat enforcers](https://github.com/MetaMask/metamask-docs/pull/reference/caveats.md) for common use cases. For more specific control, you can create custom caveat enforcers."?

Reply:
I made minor edits here, but not sure if "built-in" is the right term to use for the caveat enforcers, as it might imply that they are applied by default?